Makefile

```diff
@@ -26,9 +26,14 @@ deploy:
 	@@sqitch deploy ${TEST_DB};
 .PHONY: deploy
 
-prove:
-	# Run test suite using pg_prove
-	@@${PG_PROVE} -v -d ${TEST_DB} test/*
+prove_style:
+	# Run style-related test suite on all objects in db using pg_prove
+	@@${PG_PROVE} -v -d ${TEST_DB} test/**/*_test.sql
```

Comment:
as is, this will run all `*_test.sql` in all subfolders of `test/` due to the `/**/*` path globbing